client/@types/types.d.ts

```diff
@@ -51,13 +51,16 @@ interface IDashboardConfig extends IDataSourceContainer, IElementsContainer {
    * The category to put this template in the dashboard creation screen
    */
   category?: string,
-
+  /** 
+    * A flag indicates whether the template is featured
```

Comment:
A flag indicates whether the template is featured at the top of the dashboard creation screen